doc/release/calendar.rst

```diff
@@ -20,160 +20,201 @@ Release lifetime table
 The release lifetime table lists all Tarantool versions
 starting from 1.10.x up to the current latest versions.
 Each link leads to the release notes page of the corresponding version.
-End of series support means the release series will no longer receive any patches, updates, or feature improvements.
+*End of life* means the release series will no longer receive any patches, updates,
+or feature improvements after the specified date.
+*End of support* means that we won't provide technical support to product versions
+after the specified date.
 
 ..  container:: table
 
     ..  list-table::
 
         *   -   Version
             -   Release date
+            -   End of life
             -   End of series support
 
         *   -   :doc:`2.11.0 LTS </release/2.11.0>`
             -   May 22, 2023
+            -   May 22, 2025
```

Comment:
In the 2.11 What's new, I added the info about four years of maintenance:
```
2.11 is the long-term support (LTS) release with four years of maintenance.
```

Shouldn't we change this to `two years of maintenance`? In the release policy, I also see two  info about two years:

```
The intended support time for each series is at least two years since the first release.
```

Reply:
AFAIK, 2.11 is planned as the end of the 2nd major version and we want 4 years support for minor releases of this kind. But, please, don't lean on me here and, if you have any doubts, check it with @NeraverinTarantool and @kyukhin -- both.

Reply:
As discussed in DM: `EOL = May 2025` means that we won't release any patch versions 2.11.* after this date. Still, we're gonna provide tech support at least two years after, which makes it `four years of maintenance`.